monkey/monkey_island/cc/ui/src/components/configuration-components/UiSchema.js

```diff
@@ -123,14 +123,6 @@ export default function UiSchema(props) {
           'ui:widget': AdvancedMultiSelect
         }
       },
-      monkey: {
-        alive: {
-          classNames: 'config-field-hidden'
-        },
-        aws_keys: {
-          classNames: 'config-field-hidden'
-        }
-      },
```

Comment:
How is `config-field-hidden` different from `{'ui:widget': 'hidden'}`?
Maybe @VakarisZ knows.

Reply:
I think it refers to this [issue](https://github.com/rjsf-team/react-jsonschema-form/issues/548).